src/tests/unit/tests/injected/frameCommunicators/window-message-handler.test.ts

```diff
@@ -9,6 +9,7 @@ import { FrameMessageResponseCallback, WindowMessageHandler } from '../../../../
 import { WindowMessageMarshaller } from '../../../../../injected/frameCommunicators/window-message-marshaller';
 
 describe('WindowMessageHandlerTests', () => {
+    const starOrigin = '*' as (string & Transferable[]);
```

Comment:
What is this Transferable[] about? Should we establish a type for this; seems to be used in a few places.

Reply:
Not sure what it is, it came up from an error while updating typescript so I'm just casting to the type to fix all errors.

Reply:
`Transferable` documentation [here](https://developer.mozilla.org/en-US/docs/Web/API/Transferable)